client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/SslConfigurationBuilder.java

```diff
@@ -223,6 +224,17 @@ public SslConfigurationBuilder ciphers(String... ciphers) {
       return enable();
    }
 
+   /**
+    * Configures whether to enable hostname validation according to <a href="https://datatracker.ietf.org/doc/html/rfc2818">RFC 2818</a>.
+    * This is enabled by default and requires that the server certificate includes a subjectAltName extension of type dNSName or iPAddress.
```

Comment:
I assume we're going to disable this by default on the 14.0.x branch? This could break a lot of existing deployments.

Reply:
I need to run this by PST

Reply:
PST confirms that, because this is a CVE, it needs to be enabled by default. No way around it.